src/entities/mellowLpRouter.ts

```diff
@@ -78,6 +93,53 @@ class MellowLpRouter {
     return ethers.utils.parseUnits(amount.toString(), this.tokenDecimals);
   };
 
+  validateVaultIndices = (): boolean => {
+    if (!this.vaultIndices.every((val, i, arr) => !i || arr[i - 1] <= val)) {
```

Comment:
sure, I'll move this to a variable (this only checks if the array is sorted)

Reply:
This could be understood from the console log, but I think it makes sense to add it to a variable since we are going to remove console logs (unless we want to raise an error?)

Reply:
you can keep console logs in form of objects that you return...

perhaps you can return { error: true, message: '...' } if invalid, instead of just false

Reply:
note: this is just a helper function. It returns a boolean so that I know whether the original function called on the SDK should return early. If I return this object from the helper function, shall we return it from the original function as well? Do we plan to handle this in the UI?